File: AIRMAM5/Scripts/Models/Interface/Shared/OnlineSignalR/UserNotifyModel.ts
/** 使用者通知訊息與數量 */
export interface UserNotifyModel {
    /** 使用者userid */
    UserId: string;
    /**未讀訊息數量 */
    UnRead: number;
    /** 訊息清單(僅顯示未刪除) */
    DataList: Array<NotifyDataModel>;
}

export interface NotifyDataModel {
    /** 訊息通知編號 */
    NOTIFY_ID: number;
    /**訊息通知標題 */
    TITLE: string;
    /** 訊息通知內容 */
    CONTENT: string;
    /**是否讀取 */
    IsRead: boolean;
    /**創建日期 */
    CREATED_DATE: string;
    /**創建者 */
    CREATED_BY: string;
}

File: AIRMAM5.DBEntity/Models/Notify/UserNotifyModel.cs
using System.Collections.Generic;

namespace AIRMAM5.DBEntity.Models.Notify
{
    /// <summary>
    /// 使用者通知訊息與數量 Model
    /// </summary>
    /// <remarks> 主頁面上方快捷列: 未讀訊息,訊息列表 </remarks>
    public class UserNotifyModel
    {
        /// <summary>
        /// 使用者userid
        /// </summary>
        public string UserId { get; set; }

        /// <summary>
        /// 未讀訊息數量
        /// </summary>
        public int UnRead { get; set; } = 0;

        /// <summary>
        /// 訊息清單(僅顯示未刪除)
        /// </summary>
        public List<NotifyDataModel> DataList { get; set; } = new List<NotifyDataModel>();
    }
}
